src/bootstrap.h

```diff
@@ -7,11 +7,13 @@ class Bootstrap : public JavaInstanceWrapper {
 public:
     typedef void (*LoadClassesHook)(JNIEnv* p_env, jobject p_this, jobjectArray classes);
     typedef void (*UnloadClassesHook)(JNIEnv* p_env, jobject p_this, jobjectArray classes);
+    typedef void (*RegisterManagedEngineTypes)(JNIEnv* p_env, jobject p_this, jobjectArray classes_names);
 
     Bootstrap(jni::JObject p_wrapped, jni::JObject p_class_loader);
     ~Bootstrap() = default;
 
-    void register_hooks(jni::Env& p_env, LoadClassesHook p_load_classes_hook, UnloadClassesHook p_unload_classes_hook);
+    void register_hooks(jni::Env& p_env, LoadClassesHook p_load_classes_hook, UnloadClassesHook p_unload_classes_hook,
+                        RegisterManagedEngineTypes p_register_managed_engine_types_hook);
```

Comment:
Is it not common for cpp to format functions like this similar to kotlin?:
```cpp
void register_hooks(
    jni::Env& p_env, LoadClassesHook p_load_classes_hook,
    UnloadClassesHook p_unload_classes_hook,
    RegisterManagedEngineTypes p_register_managed_engine_types_hook
);
```

Reply:
I don't think there is "common practices" in cpp. I saw projects with methods and properties in snake case (like godot) and projects in camel case.

Reply:
Ah i don't mean the casing
I mean the formatting in terms of line breaks